webapp-django/crashstats/crashstats/utils.py

```diff
@@ -432,22 +442,19 @@ def get_version_context_for_product(product):
 
     versions = get_versions_for_product(product, use_cache=False)
 
-    # Set of X.Yb to add
-    betas = set()
-
     # Map of major version (int) -> list of (key (str), versions (str)) so
     # we can get the most recent version of the last three majors which
     # we'll assume are "featured versions"
     major_to_versions = OrderedDict()
     for version in versions:
+        # In figuring for featured versions, we don't want to include the
+        # catch-all-betas X.Yb faux version
+        if version.endswith('b'):
+            continue
```

Comment:
Liz suggested that we shouldn't include esr versions in the featured versions list. That's part of the issue in bug 1545426. If we wanted to nix esr, then we'd do that here, too.